Makefile

```diff
@@ -12,7 +12,7 @@ define make_docker
 	docker build --build-arg SVC_NAME=$(subst docker_,,$(1)) --tag=mainflux/$(subst docker_,,$(1)) -f docker/Dockerfile .
 endef
 
-all: $(SERVICES) mqtt
+all: test $(SERVICES) mqtt
```

Comment:
My idea was to run the tests on build, so that if someone changes something he/she will be notified when building the services.

Thinking twice about this, I removed the test target in the build. Let's believe in the developer discipline and running tests after each change. Even if they don't run the tests locally, the tests will be run when sending a PR, so if something is broken the PR will be marked as erroneous.

What do you think about the call of test in the dockers target? Do you also think that the tests should not be run before building the docker images and pushing them to Docker Hub?